### ISSUE TITLE
Feature Request: Global settings/templates

### ISSUE THREAD

**Issue by Orrison** [marcellino-ornelas/create-react-component#8]
Thank you so much for making this project, it has saved me a lot of time.

I just wanted to notate it could be very useful to be able to create global settings and templates. Which could be overridden by local settings/templates.

But generally I make a lot of my components similar so being able to create globals and not spend time copying over templates from another project would be awesome.

Thank you!

**Comment by marcellino-ornelas** [comment]
Hey @Orrison, 

Thank you for your feedback. Your right this would be a great feature to have and save a lot of time not trying to copy and paste from another folder. I'm currently working on a new package similar to this one but a lot more features. Currently, the architecture for this project is a little messy and only relatable to react. I'm now building, with inspiration from this project, a template system so it can be used with any framework and much more. It's currently a work in progress and hope for it to be done in the next month or so. I really like the feedback you added here and will add this feature to the new project. I will continue to update this project the best I can but im really focused on the new on im working on. I will continue to fix any bugs but new features might take longer. 

Thanks so much for your feedback!

**Comment by marcellino-ornelas** [comment]
Hey @Orrison! I hope you're doing well. I wanted to share some exciting news with you. We've just completed our new version, and we'd love for you to give it a try!

Introducing [Templates](https://marcellino-ornelas.github.io/templates/), a highly versatile package that serves a similar purpose to create-components-react. The best part is that it can be used with any framework or language, making it even more convenient.

Templates shares the same infrastructure as create-components-react, making the transition seamless. We've also included a handy migrate command, so you won't have to worry about any manual conversions. Additionally, global templates are now supported!

If you're currently using create-components-react, we've got you covered with a [useful guide](https://marcellino-ornelas.github.io/templates/docs/main/intro#coming-from-create-components-react) to help you make the switch.

We value your feedback, so feel free to share your thoughts. Contributions to the project are also welcome!

We've noticed various implementations of this concept like create-react-app, create-react-components, keystone, and many more. Our aim is to provide a reliable and comprehensive solution, and we believe it's time to consolidate efforts toward one powerful tool.

Give Templates a try at [Templates](https://marcellino-ornelas.github.io/templates/) and let us know what you think! Looking forward to your feedback and support!
